src/Augurk.Specifications/Augurk.Specifications.csproj

```diff
@@ -1,198 +1,18 @@
-    </Compile>
-    <Compile Include="Gherkin\ServerSideTags.feature.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>ServerSideTags.feature</DependentUpon>
-    </Compile>
-    <Compile Include="Portal\Versioned Features\V1\Versioning1.0.feature.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Versioning1.0.feature</DependentUpon>
-    </Compile>
-    <Compile Include="Portal\Versioned Features\V2\Versioning2.0.feature.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Versioning2.0.feature</DependentUpon>
-    </Compile>
-  </ItemGroup>
-  <ItemGroup>
-    <None Include="App.config">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="Configuration\RetentionPolicy.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>RetentionPolicy.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Documentation.md" />
-    <None Include="Gherkin\FeatureLinking.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>FeatureLinking.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Gherkin\DisplayGherkin.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>DisplayGherkin.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Gherkin\NotImplementedTag.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>NotImplementedTag.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Gherkin\ChildOfTag.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>ChildOfTag.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Gherkin\UML.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>UML.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Gherkin\ServerSideTags.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>ServerSideTags.feature.cs</LastGenOutput>
-    </None>
-    <None Include="packages.config">
-      <SubType>Designer</SubType>
-    </None>
-    <None Include="Portal\Versioned Features\V1\Versioning1.0.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>Versioning1.0.feature.cs</LastGenOutput>
-    </None>
-    <None Include="Portal\Versioned Features\V2\Versioning2.0.feature">
-      <Generator>SpecFlowSingleFileGenerator</Generator>
-      <LastGenOutput>Versioning2.0.feature.cs</LastGenOutput>
-    </None>
-  </ItemGroup>
+
   <ItemGroup>
-    <Content Include="Gherkin\FeatureLinking.png" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="MSTest.TestAdapter" Version="1.3.2" />
+    <PackageReference Include="MSTest.TestFramework" Version="1.3.2" />
+    <PackageReference Include="SpecFlow" Version="3.0.132-beta" />
```

Comment:
SpecFlow 3.0 isn't final yet, but since we aren't really automating these scenario's at all right now, I figured it wasn't a big deal to add this dependency now.

Reply:
We probably need it because there isn't a 2.x one for .NET Core? If so, this is fine. Otherwise, I prefer the usage of stable versions. Somehow referencing beta versions (even for this) seems to deter people from using open source tooling...

Reply:
Nope, earlier versions of SpecFlow do not support .NET Core. Hopefully they'll release 3.0 before we release 3.0, but I'm not entirely sure that the plan is. I guess we can re-evaluate this once we get to a point that Augurk 3.0 is actually shippable.